### ISSUE TITLE
Instructions too ambiguous

### ISSUE THREAD

**Issue by sammarcus** [learn-co-curriculum/ttt-4-display-board-rb#14]
I think the instructions in this lab are too ambiguous, and there is too large of a gap between what it explains and what the actual objective is.

I've debugged 20 or 30 students who come prepared to solve this with the previous lab's answer, but are still not sure how this is different, what needs to be done, or even how to do it.

The lab only mentions "interpolate" once, and doesn't emphasize that in actuality, that is the crucial difference and the key to solving it. It also doesn't emphasize the 3rd listed objective, or give an example of how to do so.

This results in many students stuck at square one (no pun intended) and just open a new question with 

```
def display_board(board)
  puts "   |   |   "
  puts "-----------"
  puts "   |   |   "
  puts "-----------"
  puts "   |   |   "
end
```

with no idea how to transform that into a functional answer for this.

**Comment by AnnJohn** [comment]
Reassigning to @aviflombaum

**Comment by sammarcus** [comment]
check out https://github.com/learn-co-curriculum/ttt-4-display-board-rb/pull/15 for this, it's not a hole-in-one :golf:  but I think it would improve it a bit

**Comment by sammarcus** [comment]
Could someone please review my PR? Still getting lots of questions around this :/

**Comment by aviflombaum** [comment]
Closed by f989cda

**Comment by sammarcus** [comment]
Nice, thank you very much. I like your implementation better!
